File: MediATech/Model/BL/DVD.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace MediATech.Model.BL
{
    public class DVD : MediaObject
    {
        private int _duration;


        public DVD(int id, string title, bool available, string state, DateOnly publicationDate, int duration)
            : base(id, title, available, state, publicationDate)
        {
            Duration = duration;
        }


        public int Duration { get =>  _duration; set => _duration = value;}
    }
}

File: src/main/java/mediatech/Model/BL/DVD.java
package mediatech.Model.BL;

import java.util.Date;

public class DVD extends MediaObject {

    private int duration;

    public DVD(int id, String title, boolean available, String state, Date publicationDate, int duration) {
        super(id, title, available, state, publicationDate);

        this.duration = duration;
    }

    public DVD() {}

    public int getDuration() {
        return duration;
    }
    public void setDuration(int duration) {
        this.duration = duration;
    }
    
}
